src/tests/Common/helixpublishwitharcade.proj

```diff
@@ -445,6 +443,10 @@
       <PayloadZipFile>$(MergedPayloadsRootDirectory)%(PayloadGroup).zip</PayloadZipFile>
     </MergedPayloads>
 
+    <MergedPayloads Update="@(MergedPayloads)" Condition="'$(TargetOSSpec)' != 'windows'">
+      <MergedTestHelixCommand>chmod +x %(MergedPayloads.MergedTestHelixCommand) && %(MergedPayloads.MergedTestHelixCommand)</MergedTestHelixCommand>
```

Comment:
```suggestion
      <MergedTestHelixCommand>chmod +x %(MergedPayloads.MergedTestHelixCommand) &amp;&amp; %(MergedPayloads.MergedTestHelixCommand)</MergedTestHelixCommand>
```

We need to XML-escape the ampersands.

Reply:
Hehe, thanks Jeremy, I hit the failure and updated the PR at almost exactly the same time you noticed that.

Reply:
Interestingly enough it doesn't work either.

<pre>
+ chmod +x JIT/Methodical/Arrays/range/range.sh
/tmp/helix/working/A7D0094C/p/scripts/8c01f305203a4a41ab3bec438cafdfff/execute.sh: line 16: syntax error near unexpected token `&&'
/tmp/helix/working/A7D0094C/p/scripts/8c01f305203a4a41ab3bec438cafdfff/execute.sh: line 16: ` && JIT/Methodical/Arrays/range/range.sh'
</pre>

As the next attempt I'm trying to use a two-line Helix command, I have yet to see why that fails.